addon/tests/helpers/input.js

```diff
@@ -9,52 +16,48 @@ var isHidden = function(selector) {
 };
 
 var isTextInput = function(selector) {
-    var element = find(selector);
-    equal(element.length, 1);
-    equal(element.attr("type"), "text");
+    assertElement(selector, "text");
 };
 
 var isEmailInput = function(selector) {
-    var element = find(selector);
-    equal(element.length, 1);
-    equal(element.attr("type"), "email");
+    assertElement(selector, "email");
 };
 
 var isPasswordInput = function(selector) {
-    var element = find(selector);
-    equal(element.length, 1);
-    equal(element.attr("type"), "password");
+    assertElement(selector, "password");
 };
 
 var isTelInput = function(selector) {
-    var element = find(selector);
-    equal(element.length, 1);
-    equal(element.attr("type"), "tel");
+    assertElement(selector, "tel");
 };
 
-var isHiddenInput = function(name, value) {
-    var element = find("input[name=%@]".fmt(name));
-    equal(element.length, 1);
-    equal(element.attr("type"), "hidden");
-    equal(element.val(), value);
+var isCheckbox = function(selector) {
+    assertElement(selector, "checkbox");
 };
 
-var isCheckbox = function(selector) {
-    var element = find(selector);
-    equal(element.length, 1);
-    equal(element.attr("type"), "checkbox");
+var isHiddenInput = function(name, value) {
+    var element = assertElement("input[name=%@]".fmt(name), "hidden");
+    equal(element.val(), value, "value of '" + value + "' is not equal to the element's value of '" + element.val() + "'");
 };
 
 var isRadioButton = function(selector, count) {
-    var element = find(selector);
-    equal(element.length, count || 1);
-    equal(element.attr("type"), "radio");
+    assertElement(selector, "radio", count);
+};
+
+var isTextButton = function(selector, text, type) {
+    if (!text) {
+        ok(false, "Text must be passed to the isTextButton assertion function");
```

Comment:
this *might* be better as Ember.assert like I did here (not sure it matters really -just seen the Ember.assert used in core a lot)

https://github.com/toranb/ember-cli-simple-store/blob/master/addon/store.js#L62